modules/azure/src/main/java/org/testcontainers/containers/AzuriteContainer.java

```diff
@@ -0,0 +1,529 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Testcontainers implementation for Azurite Emulator.
+ * <p>
+ * Supported image: {@code mcr.microsoft.com/azure-storage/azurite}
+ * <p>
+ * Exposed ports:
+ * <ul>
+ *     <li>10000 (default blob port, configurable)</li>
+ *     <li>10001 (default queue port, configurable)</li>
+ *     <li>10002 (default table port, configurable)</li>
```

Comment:
not configurable. TBH, I don't see value customizing container ports, host ports are already random so no conflict ports is produced.

Reply:
Thank you! I will remove all port configuration related functionality.

Reply:
Done, thank you!